sdk/synapse/azure-analytics-synapse-spark/src/main/java/com/azure/analytics/synapse/spark/SparkServiceVersion.java

```diff
@@ -1,36 +0,0 @@
-// Copyright (c) Microsoft Corporation. All rights reserved.
-// Licensed under the MIT License.
-
-package com.azure.analytics.synapse.spark;
-
-import com.azure.core.util.ServiceVersion;
-
-/**
- * The versions of Azure Synapse Analytics Access Control supported by this client library.
- */
-public enum SparkServiceVersion implements ServiceVersion {
```

Comment:
Any reason for removing this? This is a handwritten class and is part of the public API which should be included in the builder.

Reply:
Discussed offline, this is no longer required as the library is fully autogenerated and the builder has a method to take the service version as a string.

Reply:
Synced offline - since this is a completely autogenerated library, removing this class for now. The breaking change is documented in the change log.